### DISCUSSION TITLE
Option to force selection of confirmed utxos only [Greenfields API]

### DISCUSSION THREAD

**Discussion by danielalexiuc** [btcpayserver/btcpayserver#3737] [Technical Support]
Greenfields API: Create store on-chain wallet transaction endpoint
/api/v1/stores/{storeId}/payment-methods/onchain/{cryptoCode}/wallet/transactions

It would be good to have an option to force the selection of CONFIRMED utxos only on this endpoint. 

Currently, the endpoint will select random utxos that may or may not be confirmed. In high fee environments , selecting unconfirmed utxos can result in unintentional chains of unconfirmed CPFP transactions.

**Comment by bolatovumar** [comment]
Good idea @danielalexiuc! PR to implement this here: https://github.com/btcpayserver/btcpayserver/pull/3850
